src/main/kotlin/org/rust/lang/template/RustFileContextType.kt

```diff
@@ -6,7 +6,7 @@ import com.intellij.psi.util.PsiUtilCore
 import org.rust.lang.RustLanguage
 
 
-class RustFileContextType : TemplateContextType("RUST_FILE", "Rust file") {
```

Comment:
I'm not sure about this. `Rust` by itself is good, but we will also want `Rust expression`, `Rust module` soon (In fact, we can already implement them) soon.